src/ngraph/runtime/intelgpu/intelgpu_tensor_view.hpp

```diff
@@ -38,7 +38,8 @@ class ngraph::runtime::intelgpu::IntelGPUTensorView : public ngraph::runtime::Te
     IntelGPUTensorView(const element::Type& element_type,
                        const Shape& shape,
                        const cldnn::engine& backend_engine,
-                       void* memory_pointer = nullptr);
+                       void* memory_pointer,
```

Comment:
Is it necessary to eliminate default parameter?

Reply:
Strictly no, but I added a new parameter that I don't want to have a default. I could have added this new parameter between backend_engine and memory_pointer but that would have change the signature more than I wanted. I decided to remove the default parameter as this constructor is called in exactly two places, both within private parts of the intel gpu backend so the default parameter seems unnecessary. The backend parameter is required and therefore can't be optional and you can't have a required parameter after an optional one. The change seemed harmless but if you want I can change to something you suggest.